Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -68,21 +69,54 @@ click: function () { closeClicked.raiseEvent(this); }');
         closeElement.innerHTML = '&times;';
         infoElement.appendChild(closeElement);
 
-        var infoBodyElement = document.createElement('div');
-        infoBodyElement.className = 'cesium-infoBox-body';
-        infoElement.appendChild(infoBodyElement);
-
-        var descriptionElement = document.createElement('div');
-        descriptionElement.className = 'cesium-infoBox-description';
-        descriptionElement.setAttribute('data-bind', '\
-html: descriptionSanitizedHtml,\
-style : { maxHeight : maxHeightOffset(40) }');
-        infoBodyElement.appendChild(descriptionElement);
+        var frame = document.createElement('iframe');
+        frame.className = 'cesium-infoBox-iframe';
+        frame.setAttribute('data-bind', 'style : { maxHeight : maxHeightOffset(40) }');
+        frame.setAttribute('sandbox', 'allow-same-origin allow-popups allow-pointer-lock allow-forms'); // allow-scripts allow-top-navigation
+        frame.setAttribute('allowfullscreen', true);
+        infoElement.appendChild(frame);
 
         var viewModel = new InfoBoxViewModel();
+        knockout.applyBindings(viewModel, infoElement);
+
+        //We inject default css into the content iframe,
+        //end users can remove it or add their own via the exposed frame property.
+        var cssLink = document.createElement("link");
+        cssLink.href = buildModuleUrl('Widgets/InfoBox/InfoBoxDescription.css');
+        cssLink.rel = "stylesheet";
+        cssLink.type = "text/css";
+
+        //div to use for description content.
+        var frameContent = document.createElement("div");
+        frameContent.className = 'cesium-infoBox-description';
+
+        this._container = container;
+        this._element = infoElement;
+        this._frame = frame;
         this._viewModel = viewModel;
+        this._processedDescriptionSubscription = undefined;
+
+        var that = this;
+
+        //We can't actually add anything into the frame until the load event is fired
+        frame.addEventListener('load', function() {
+            var frameDocument = frame.contentDocument;
+            frameDocument.head.appendChild(cssLink);
+            frameDocument.body.appendChild(frameContent);
+
+            //We manually subscribe to the
+            that._processedDescriptionSubscription = subscribeAndEvaluate(viewModel, 'processedDescription', function(value) {
+                frameContent.innerHTML = value;
+                frame.style.height = frameContent.getBoundingClientRect().height + 'px';
 
-        knockout.applyBindings(this._viewModel, infoElement);
+                //Chrome seems to get the height wrong above and requires us to meaure the body
+                //after setting the frame. Other browsers work without the below line.
+                frame.style.height = frameDocument.body.getBoundingClientRect().height + 'px';
```

Comment:
Okay, I'll need some help here.  

1. Comment out line 114 and bring up simple.czml in Chrome.
2. Click on the ISS track (purple line)
3. Click on the AGI facility (do not click on anything else in between or it doesn't work).

The InfoBox will be the wrong size with a scroll bar added when it shouldn't be.  If you close the info box and click on AGI again, the problem isn't there.  This does not happen in other browsers.  It also looks like Chrome is sticking extra space in the iframe body after the div, and I have no idea why.